tests/test_utils.py

```diff
@@ -138,6 +138,39 @@ def test_standardize_values_TypeError(self):
             assert "Error comparing types." in captured_output.getvalue()
             assert standard_df.equals(self.df)
 
+    def test_standardize_values_preserves_n_a_substrings(self):
+        """Test that N/A substrings within other text are not accidentally replaced."""
+        df_with_substrings = pd.DataFrame(
+            {
+                "aliases": [
+                    "Snx1*D465N/APOE4/Trem2*R47H",  # Contains "N/A" as substring
+                    "Trem2*R47H",  # No N/A
+                    "APOE4*N/A*Trem2",  # Contains "N/A" as substring
+                    "N/A",  # Exact N/A - should be replaced
+                    "n/a",  # Exact n/a - should be replaced
+                    "Some text with N/A in it",  # Contains "N/A" as substring
+                    "Normal text",  # No N/A
```

Comment:
Future improvement: These are technical different test cases, that you could leverage `pytest.parametrize` functionality.  You can imagine each test function being split to the particular scenario and each scenario having different set of parameters and test input.

```
def test_standardize_values_nas_removed
def test_standardize_values_nas_preserved
def test_standardize_values_no_nas
```

May be overkill for this particular scenario, but just a thought.